configure.ac

```diff
@@ -599,6 +599,18 @@ then
   AM_CFLAGS="-DWOLFSSL_DTLS_MTU $AM_CFLAGS"
 fi
 
+# DTLS resend last flight only on a timeout
```

Comment:
Please remove this from configure.ac and put a note about the new flag with the others at the top of src/internal.c.